modeling-cmds/src/websocket.rs

```diff
@@ -93,11 +93,25 @@ pub enum WebSocketRequest {
 }
 
 /// A sequence of modeling requests. If any request fails, following requests will not be tried.
-#[derive(Serialize, Deserialize, JsonSchema, Debug, Default)]
+#[derive(Serialize, Deserialize, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 pub struct ModelingBatch {
     /// A sequence of modeling requests. If any request fails, following requests will not be tried.
     pub requests: Vec<ModelingCmdReq>,
+    /// ID of batch being submitted.
+    /// Each request has their own individual ModelingCmdId, but this is the
+    /// ID of the overall batch.
+    pub id: ModelingCmdId,
```

Comment:
```suggestion
    pub cmd_id: ModelingCmdId,
```

made consistent w the others

Reply:
I deliberately wanted it different because it's the ID of the batch, not of any particular modeling command.